proxy/prefect_flows.py

```
@@ -5,6 +5,7 @@
 """
 
 import os
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)